### PR TITLE
Move articles about the third-party tools and libs

### PR DESCRIPTION
**Pull request by jsconan**
Move articles related to the third party tools and libraries from the backup of the forge wiki to the articles

### CODE REVIEW

File: articles/Third Party Tools and Libraries/enable-math.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!--
-parent: Wiki
+parent: articles
```
Comment (dietertaotesting) [inline articles/Third Party Tools and Libraries/enable-math.md]: Parent is the category under which an article is listed e.g. on https://hub.taocloud.org/articles (Administrator Guide, Data Abstractions etc.). If set at all it should be in proper spelling, not lower case etc. On the other hand you can ommit it. In this case the directory name will be used.
Example:
taohub-articles/articles/Some Category/my-article.md (assumed the title is My Article)
will be displayed as
**Some Category**
  - My Article

File: articles/Third Party Tools and Libraries/enable-math.md [hunk 2]
```diff
@@ -9,21 +9,32 @@ contributors:
...
 tags:
-    - Wiki
+    - 'Frontend'
+    - 'Library'
+    - 'JavaScript'
+    - 'Math'
+    - 'MathJax'
+    - 'MathML'
+    - 'LaTex'
+    - 'ThirdParty'
```
Comment (dietertaotesting) [inline articles/Third Party Tools and Libraries/enable-math.md]: Tags are mainly used to build the navigation from https://hub.taocloud.org/taohubdocs/tags-and-tag-groups, not so much as keywords for search. Insofar ThirdParty or Library are not ideal

File: articles/Third Party Tools and Libraries/enable-math.md [hunk 3]
```diff
@@ -47,7 +58,7 @@ Frist delete the following folders:
 
 Then in the `mathjax/config/` folder, delete all files but `TeX-AMS-MML_HTMLorMML-full.js`, which is the only config file used in TAO.
 
-An already shrunk Mathjax archive is available [here](../resources/mathjax-shrinked.zip).
+An already shrunk Mathjax archive is available [here](../resources/third-party/mathjax-shrinked.zip).
```
Comment (dietertaotesting) [inline articles/Third Party Tools and Libraries/enable-math.md]: Spelling: mathjax-shrinked should also be shrunk (http://www.verbix.com/webverbix/English/shrink.html)

File: articles/Third Party Tools and Libraries/enable-math.md [hunk 4]
```diff
@@ -9,21 +9,32 @@ contributors:
...
+    - 'ThirdParty'
 -->
 
 # Enable math expression in items
 
-![](../resources/badge-square.png)
+![logo](../resources/third-party/badge-square.png)
+
+> MathJAX is a library for rendering Math expressions within a web page.
+> It supports both MathML and LaTex expressions.
```
Comment (dietertaotesting) [inline articles/Third Party Tools and Libraries/enable-math.md]: The combination title -> synopsis as one line -> logo is well tested, this order not so much. I'm pretty curious to see if this passes my parser with the expected result. We'll see :-)

File: articles/Third Party Tools and Libraries/pdf-printing.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+![wkhtmltopdf logo](../resources/third-party/wkhtmltopdf.png) https://wkhtmltopdf.org/
+
+> wkhtmltopdf is open source (LGPLv3) command line tool to render HTML into PDF using the Qt WebKit rendering engine.
+> This run entirely "headless" and do not require a display or display service.
+
+An extension for TAO allow to create test booklets: [taoBooklet](https://github.com/oat-sa/extension-tao-booklet).
+However this extension needs a third-party tool to generate the PDF files.
+So to be able to generate the booklet, you should install `wkhtmltopdf` on your server.
+
```
Comment (dietertaotesting) [inline articles/Third Party Tools and Libraries/pdf-printing.md]: You could mention, that there is also a binary for Windows on https://wkhtmltopdf.org/downloads.html, but that this has not been tested by us.

File: forge/QTI-in-TAO.md [hunk 6]
```diff
@@ -69,7 +69,7 @@ The item body support images, audio and video. Images are inserted as simple *ht
 
 The supported video formats are mpeg4 (H.264+AAC) and *youtube*. We suggest using mp3 audio format to ensure cross-browser compatibility.
 
-Math expressions are also available. It however requires the [installation of a free third-party library MathJax](http://forge.taotesting.com/projects/tao/wiki/Enable_math).
+Math expressions are also available. It however requires the [installation of a free third-party library MathJax](https://github.com/oat-sa/taohub-articles/blob/master/articles/Third%20Party%20Tools%20and%20Libraries/enable-math.md).
 
 As to PDF files, the issue is the same as math expressions and multimedia. There is no way to ensure cross-browser compatibility natively. Currently, pdf files are therefore not fully supported in TAO QTI items. For more information, please see the open issue <br/>
```
Comment (dietertaotesting) [inline forge/QTI-in-TAO.md]: what ist this ref to #2503? Is the link correct, also is the <br> required?

**Review by ghost — CHANGES_REQUESTED** [review]
Please read the comments in https://github.com/oat-sa/taohub-articles/pull/21/files#diff-24a08789514e82e23208f463a3adc88a first as they apply to other files too

**Comment by jsconan** [comment]
Changes made

**Review by ghost — APPROVED** [review]

**Pull request merged**